### ISSUE TITLE
Broken link

### ISSUE THREAD

**Issue by zmaril** [clojure-grimoire/grimoire#166]
http://conj.io/store/v0/org.clojure/clojurescript/0.0-2498/clj/cljs.util/path-seq

**Comment by arrdem** [comment]
Yeah ClojureScript support got dropped for a while pending a boatload of work on the documentation generator tool. Sorry about that.

**Comment by arrdem** [comment]
Update: ClojureScript data would now be fully supported. However no such data exists. lein-grim needs to be updated so that it can kick out cljs and clj data files and then both need to be generated for cljs. Since Shaun et all seem to have things well under hand with their cljs quickref this isn't a high priority item.
